packages/plugin-e2e/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@grafana/plugin-e2e",
-  "version": "0.3.1",
-  "private": true,
+  "version": "0.0.0",
```

Comment:
let's publish 0.0.1?

Reply:
Since the `patch` label is added, the version will be bumped to 0.0.1 before publishing to npm.